pkgs/development/python-modules/mypy/extensions.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchPypi, buildPythonPackage, typing, isPy3k }:
+
+buildPythonPackage rec {
+  pname = "mypy_extensions";
+  version = "0.4.1";
+
+  # Tests not included in pip package.
+  doCheck = false;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "04h8brrbbx151dfa2cvvlnxgmb5wa00mhd2z7nd20s8kyibfkq1p";
+  };
+
+  disabled = !isPy3k;
```

Comment:
Just noticed: I think this is enabled for python2, no? https://github.com/python/mypy/blob/master/extensions/setup.py#L21

Reply:
080b3ef0b59